cluster-microservice-provider/src/main/java/io/silverware/microservices/providers/cluster/internal/JgroupsMessageSender.java

```diff
@@ -47,8 +48,8 @@
     */
    private static final Logger log = LogManager.getLogger(JgroupsMessageSender.class);
 
-   private static final RequestOptions SYNC_OPTIONS = RequestOptions.SYNC();
-   private static final RequestOptions ASYNC_OPTIONS = RequestOptions.ASYNC();
+   private static final RequestOptions SYNC_OPTIONS = new RequestOptions(ResponseMode.GET_ALL, 10000, true);
+   private static final RequestOptions ASYNC_OPTIONS = new RequestOptions(ResponseMode.GET_NONE, 10000, true);
```

Comment:
Can the values passed to `RequestOptions` constructor be moved to constants with self explanatory names in order to easily see what they mean?